nitrite-mvstore-adapter/src/main/resources/META-INF/native-image/org.dizitart/nitrite-mvstore-adapter/reflect-config.json

```diff
@@ -0,0 +1,50 @@
+[
```

Comment:
hints for reflective constructor calls in `org.dizitart.no2.mvstore.compat.v1.mvstore.fs.FilePath#registerDefaultProviders`